modules/claims_api/lib/claims_api/v2/disability_compensation_pdf_mapper.rb

```diff
@@ -100,6 +102,34 @@ def conditions_related_to_exposure?
         # This will check 'YES' for box 15A.
         # TODO: for disability mapping ticket
       end
+
+      def treatment_centers
+        @pdf_data[:data][:attributes][:claimInformation] = {}
+        @pdf_data[:data][:attributes][:claimInformation].merge!(
+          treatments: []
+        )
+        treatments = get_treatments
+
+        treatment_details = treatments.map(&:deep_symbolize_keys)
+        @pdf_data[:data][:attributes][:claimInformation][:treatments] = treatment_details
+
+        @pdf_data
+      end
+
+      def get_treatments
+        # Concat treatmentDetails as: Disability Name - Treatment Center Name, City, State
+        @auto_claim['treatments'].map do |tx|
+          center = "#{tx['center']['name']}, #{tx['center']['city']}, #{tx['center']['state']}"
+          details = "#{tx['treatedDisabilityName']}, #{center}"
+          tx['treatmentDetails'] = details
+          tx['dateOfTreatment'] = tx['startDate'] || nil
+          tx['doNotHaveDate'] = tx['startDate'] ? false : true
```

Comment:
this is interesting to me. firefly's example request has the below
```
        {
          "treatmentDetails": "string",
          "dateOfTreatment": "2023-03-29",
          "doNotHaveDate": true
        }
```
this makes me think they always want both values and I think their example is just a bad example, because it has a start date but `doNotHaveDate` is true, so? I'm gonna say that's just a bad schema example.


also, couldn't we do 
```suggestion
          tx['dateOfTreatment'] = tx['startDate']
          tx['doNotHaveDate'] = tx['startDate'].nil?
```